streams/src/main/resources/common/message/SubscriptionInfoData.json

```diff
@@ -48,6 +48,11 @@
       "versions": "2+",
       "type": "bytes"
     },
+    {
+      "name": "shutdownRequested",
+      "versions": "8+",
+      "type": "int32"
```

Comment:
32 bits is a lot of possible error codes. Note: these numbers are encoding enum values. According to the java spec, you can't have more than 2^16 members a class. It looks like some people actually experimented with how many enum values they could add before the compiler fails, and it tops out at 2746 values. (https://stackoverflow.com/questions/4468388/maximum-number-of-enum-elements-in-java)

Either way, an int16 (`short`) should be more than enough codes.